packages/fx-core/src/plugins/solution/fx-solution/utils/progressHelper.ts

```diff
@@ -0,0 +1,28 @@
+import { IProgressHandler, PluginContext } from "../../../../../../api/build";
```

Comment:
maybe we need to reference from teamsfx-api package?

Reply:
Fixed.